packages/offix-scheduler/src/store/OfflineStore.ts

```diff
@@ -1,7 +1,8 @@
 import { PersistentStore, PersistedData } from "./PersistentStore";
 import { QueueEntryOperation, QueueEntry } from "../queue";
 import { OfflineStoreSerializer, DefaultOfflineSerializer } from "./OfflineStoreSerializer";
-
+import Debug from "debug";
```

Comment:
It is small letter

Reply:
I saw it on a stack overflow post [https://stackoverflow.com/questions/47724165/how-to-use-the-debug-module-with-typescript](post), but I'll try it with small letter too

Reply:
Stack overflow :)

Reply:
and the yarn lint-fix is not running, should I just externally remove that space?

Reply:
@wtrocki

Reply:
There are other things to fix. spaces are least concern really. VScode eslint will help as well

Reply:
You could change the import to:

`import debug from 'debug'`

And then call the variable something like `info` or `log`. I prefer info, feels more descriptive to me.

So:

`const info = debug(...)`